### ISSUE TITLE
[A11y] 1.4.13 Content on Hover or Focus - Hover Content is not Hoverable and Dismissable

### ISSUE THREAD

**Issue by bsaadat** [PublicHealthEngland/coronavirus-dashboard#95]
Tester has observed that when viewing the graphs, content that appears on hover or focus cannot be dismissed. It is expected that the user is able to dismiss content without moving pointer hover or keyboard focus, such as pressing escape on a keyboard.

Additionally to this the hover content is not persistent. It is expected that content presented through hover is persistent and is not removed until the focus trigger is removed, the user dismisses it or its information is no longer valid. User should be able to mouse over the hover content to follow the content without their mouse obscuring it.

**This does not comply with WCAG Level AA 2.1:
1.4.13 Content on Hover or Focus** - https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html 
 
**Steps to Recreate:**
1.    Navigate to graphs
2.    Hover and attempt to dismiss content or mouse over hover content
3.    Observe issue
 
**This has been observed on the following pages:**
Coronavirus (COVID-19) cases in the UK

**Environment:**
Windows 10 – Chrome 80
 
**This issue is known to occur on:**
Believed to be global
 
**Reproduction rate:**
100% - Consistently reproducible. Occurs every time that it is attempted.
 
**Supporting material:**
Screenshot/Video

**Comment by bsaadat** [comment]
![image](https://user-images.githubusercontent.com/27902577/80507257-93f5f880-896e-11ea-8fa8-b8f4a7a8e175.png)

**Comment by bsaadat** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[A11y] 1.4.13 Content on Hover or Focus - Hover Content is not Hoverable and Dismissable](https://trello.com/c/ce344vvM/222-a11y-1413-content-on-hover-or-focus-hover-content-is-not-hoverable-and-dismissable)
